aamt_parse2.pm

```diff
@@ -4908,21 +5114,24 @@ sub pars_AskSelectSites
     }
 
     $array[$sIndex][MYSQL] = 0;
-    my $promptyn;    
```

Comment:
You need to add a cmd line flag for running in unattended/non-interactive mode. If this flag is true, then you can skip this, if it is false, then keep the current code.